File: App/src/main/java/it/gruppoinfor/home2work/fragments/ProfileFragmentDashboard.java
package it.gruppoinfor.home2work.fragments;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.FrameLayout;

import java.util.ArrayList;

import butterknife.ButterKnife;
import butterknife.Unbinder;
import it.gruppoinfor.home2work.R;
import it.gruppoinfor.home2workapi.model.Share;

public class ProfileFragmentDashboard extends Fragment {

    private Unbinder mUnbinder;

    public ProfileFragmentDashboard() {
        // Required empty public constructor
    }


    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        FrameLayout root = (FrameLayout) inflater.inflate(R.layout.fragment_profile_dashboard, container, false);
        mUnbinder = ButterKnife.bind(this, root);
        initUI();
        return root;
    }

    private void initUI() {

    }

    @Override
    public void onDestroyView() {
        mUnbinder.unbind();
        super.onDestroyView();
    }

}

File: App/src/main/java/it/gruppoinfor/home2work/fragments/ProfileFragmentDashboard.kt
package it.gruppoinfor.home2work.fragments

import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.FrameLayout

import java.util.ArrayList

import butterknife.ButterKnife
import butterknife.Unbinder
import it.gruppoinfor.home2work.R
import it.gruppoinfor.home2workapi.model.Share

class ProfileFragmentDashboard : Fragment() {

    private var mUnbinder: Unbinder? = null


    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        val root = inflater.inflate(R.layout.fragment_profile_dashboard, container, false) as FrameLayout
        mUnbinder = ButterKnife.bind(this, root)
        initUI()
        return root
    }

    private fun initUI() {

    }

    override fun onDestroyView() {
        mUnbinder!!.unbind()
        super.onDestroyView()
    }

}// Required empty public constructor
